rsconf/component/sirepo.py

```diff
@@ -57,7 +57,7 @@ def internal_build_compile(self):
                 feature_config=PKDict(
                     api_modules=[],
                     job=True,
-                    default_proprietary_sim_types=tuple(),
+                    moderated_sim_types=tuple(),
```

Comment:
I think this needs to be backward compatible so we can use rsconf during the transition from alpha to prod. If default_proprietary_sim_types is set, use old behavior (Authenticator) otherwise use use new behavior (SirepoAuthenticator).

Reply:
https://github.com/radiasoft/rsconf/pull/183/commits/72f514ded748f8ad85c8781c3da339dda93c2f5a